HttpStress fails to build msquic

Starting from 02/04 the "Build HttpStress" step fails to build msquic:

https://dev.azure.com/dnceng-public/public/_build/results?buildId=160889&view=logs&j=22fea640-1099-5f32-ec5d-316ad83f359a&t=7fe64d4e-bb7a-5b68-0c5c-ad57d12d4da9

```
[ 82%] Building C object src/platform/CMakeFiles/platform.dir/platform_posix.c.o
/msquic/msquic/src/msquic/src/platform/platform_posix.c:40:10: fatal error: numa.h: No such file or directory
   40 | #include <numa.h>
      |          ^~~~~~~~
compilation terminated.
gmake[2]: *** [src/platform/CMakeFiles/platform.dir/build.make:160: src/platform/CMakeFiles/platform.dir/platform_posix.c.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:272: src/platform/CMakeFiles/platform.dir/all] Error 2
gmake: *** [Makefile:160: all] Error 2
The command '/bin/sh -c cd msquic/src/msquic &&     mkdir build &&     cmake -B build -DCMAKE_BUILD_TYPE=Release -DQUIC_ENABLE_LOGGING=false -DQUIC_USE_SYSTEM_LIBCRYPTO=true -DQUIC_BUILD_TOOLS=off -DQUIC_BUILD_TEST=off -DQUIC_BUILD_PERF=off &&     cd build &&     cmake --build . --config Release' returned a non-zero code: 2
Service 'server' failed to build : Build failed
```

@ManickaP any suggestion how to fix this?

--- a/src/libraries/System.Net.Http/tests/StressTests/HttpStress/Dockerfile
+++ b/src/libraries/System.Net.Http/tests/StressTests/HttpStress/Dockerfile
@@ -5,7 +5,7 @@ FROM $SDK_BASE_IMAGE
 WORKDIR /msquic
 RUN apt-get update -y && \
     apt-get upgrade -y && \
-    apt-get install -y cmake clang ruby-dev gem lttng-tools libssl-dev && \
+    apt-get install -y cmake clang ruby-dev gem lttng-tools libssl-dev libnuma-dev && \
     gem install fpm
 RUN git clone --recursive https://github.com/dotnet/msquic
 RUN cd msquic/src/msquic && \

